codegenerator/cli/src/rescript_types.rs

```diff
@@ -653,11 +653,23 @@ impl Serialize for RescriptTypeIdent {
     }
 }
 
+pub mod helpers {
+    use super::*;
+    pub fn option(typ: RescriptTypeIdent) -> RescriptTypeIdent {
+        RescriptTypeIdent::Option(Box::new(typ))
+    }
+
+    pub fn array(typ: RescriptTypeIdent) -> RescriptTypeIdent {
+        RescriptTypeIdent::Array(Box::new(typ))
+    }
+}
```

Comment:
I think this should be on `RescriptTypeIdent` impl. Like `RescriptTypeIdent::option`. I usually prefer to avoid helpers/utils modules, because they often turn to a messy pile of code.